jsonschema-core.xml

```diff
@@ -2570,6 +2474,124 @@
             </section>
         </section>
 
+        <section title="A Vocabulary for Unevaluated Locations">
+            <t>
+                Meta-schemas that do not use "$vocabulary" SHOULD be considered to
+                require this vocabulary as if its URI were present with a value of true.
+            </t>
+            <t>
+                The current URI for this vocabulary, known as the Unevaluated Applicator
+                vocabulary, is:
+                &lt;https://json-schema.org/draft/2019-09/vocab/unevaluated&gt;.
```

Comment:
shouldn't this be draft/2020-XX/..., or will that be changed later?

Reply:
We'll fix it all at once later.  People (possibly me, I don't recall) pushed for it earlier last time and I had to keep changing the XX and it was a giant waste of time.